README.md

```diff
@@ -102,17 +103,21 @@ When the Jupyter notebook is loaded and the kernel is ready then we can start ex
 
 ![Notebook loaded](doc/source/images/wml-3-notebook-loaded.png)
 
+> **Important**: *Make sure that you stop the kernel of your notebook(s) when you are done, in order to prevent leaking of memory resources!*
+
+![Stop kernel](doc/source/images/JupyterStopKernel.png)
+
 ### 4. Import dataset into the notebook
 
 Spend a minute looking through the sections of the notebook to get an overview.
 
 In section `2.0 Load and Clean data`, highlight the cell labelled `# Place cursor ...` by clicking on it. Click on the *10/01* button to select a specific data set.
 
-If you are following the [Learning Path](), Choose The *Remote* tab, and pick the virtualized data set that has all three joined tables (i.e. `User999.billing+products+customers`), and choose `Insert Pandas DataFrame`.
+If you are following the [CP4D Learning Path](https://developer.ibm.com/series/cloud-pak-for-data-learning-path/), Choose The *Files* tab, and pick the virtualized data set that has all three joined tables (i.e. `User<xyz>.BILLINGPRODUCTSCUSTOMERS`), and choose `Insert Pandas DataFrame`.
 
 ![Add remote Pandas DataFrame](doc/source/images/wml-4-add-dataframe.png)
 
-If you are running this pattern alone, choose the *Local* tab, and choose the 'Telco-Customer-Churn.csv` file that you added, then choose `Insert Pandas DataFrame`:
+If you are running this pattern alone, choose the *Files* tab, and choose the 'Telco-Customer-Churn.csv` file that you added, then choose `Insert Pandas DataFrame`:
```

Comment:
The single quote in front of Telco-Cust needs to be a back-tick.  The formatting is messed up.

Reply:
done